src/app/help/help.component.ts

```diff
@@ -1,6 +1,7 @@
 import { Component, ViewChild, OnInit } from '@angular/core';
 import * as $ from 'jquery';
 import { ModalComponent } from 'ng2-bs3-modal/ng2-bs3-modal';
+import Swiper from 'swiper';
```

Comment:
Could you please explain this point.

Reply:
@sakshee-19 it's because when I tried to cover this `Help Component` with tests, I got this error 
`Failed: Swiper is not defined
 ReferenceError: Swiper is not defined` 
I explored and found out this was a way to solve it. Fell free to share if you think we need to use something better to counter that error